tests/pages/pocket/base.py

```diff
@@ -42,14 +44,16 @@ def URL_TEMPLATE(self):
 
     @property
     def navigation(self):
-        return self.Navigation(self)
+        if self.is_element_displayed(*self._platform_nav_locator):
```

Comment:
Not sure this is the right way to go about it, but some base templates override the provided base header/footer, so I wanted to allow the base test template to handle both.

Reply:
seems pretty sensible to me!